### PR TITLE
[Bugfix][822]lower_case_idcams_utility

### LINKED ISSUE #822: [Bug] (Maybe) `module_utils.mvs_cmd.idcams` passes lower-case characters to the job
### Is there an existing issue for this?

- [X] There are no existing issues.

### Are the dependencies a supported version?

- [X] The dependencies are supported.

### IBM Z Open Automation Utilities

v1.2.3

### IBM Enterprise Python

v3.9.x

### IBM z/OS Ansible core Version

v1.6.0-beta.1

### ansible-version

v2.14.x

### z/OS version

v2.3

### Ansible module

_No response_

### Bug description

Not a definite bug here, but it's behaviour that may trip up users and would be good to have a conversation about it.

We're using the `module_utils.mvs_cmd.idcams` utility as part of new modules in the CICS collection.

If you pass lower-case characters into the IDCAMS script, they get converted into dots (or some other character that gets rendered as dots) and then this results in failures or incorrect behaviour.

For example, the following task results in the attach verbose log:

```yaml
    - name: Initialise a CICS global catalog with mixed data set name
      environment: "{{ environment_vars }}"
      ibm.ibm_zos_cics.global_catalog:
        state: initial
        sdfhload: "CTS560.CICS730.SDFHLOAD"
        location: "BENCOX.REGIONS.mixed.DFHGCD"
        space_primary: 5
        space_type: "M"
```

Note the (hard-coded) use of lower-case `mixed` in the data set name. We actually tripped over this problem when passing variables in for qualifiers of the data set - so it's not always obvious that the data set contains lower-case in the playbook. We were able to work around it in the playbook by using the `upper` filter. We _could_ work around it in our module.

We believe the behaviour is base IDCAMS - see [here](https://www.ibm.com/docs/en/zos/2.1.0?topic=commands-):
> The defaulted character set does not contain lower case. See [PARM Command](https://www.ibm.com/docs/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.idai200/parm.htm#parm) for information on changing the character set used by IDCAMS.

Our question is whether `module_utils.mvs_cmd.idcams` should upper-case its SYSIN input to avoid this problem occurring. Would be interested to hear your thoughts.

### Playbook verbosity output.

TASK [Initialise a CICS global catalog with mixed data set name] *****************************************************************************************************************************************************************************************************************************************************************
task path: /Users/ben/Downloads/try-ibm_zos_cics-1.1.0-beta.1/playbooks/idcams.yaml:32
<28> ESTABLISH SSH CONNECTION FOR USER: None
<28> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o 'ControlPath="/Users/ben/.ansible/cp/d4e83c6eea"' 28 '/bin/sh -c '"'"'echo ~ && sleep 0'"'"''
<28> (0, b'/u/bencox\n', b'OpenSSH_9.0p1, LibreSSL 3.3.6\r\ndebug1: Reading configuration data /Users/ben/.ssh/config\r\ndebug1: /Users/ben/.ssh/config line 1: Applying options for *\r\ndebug1: /Users/ben/.ssh/config line 31: Applying options for 28\r\ndebug1: /Users/ben/.ssh/config line 41: Applying options for *\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 21: include /etc/ssh/ssh_config.d/* matched no files\r\ndebug1: /etc/ssh/ssh_config line 54: Applying options for *\r\ndebug1: Authenticator provider $SSH_SK_PROVIDER did not resolve; disabling\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 2852\r\ndebug3: mux_client_request_session: session request sent\r\ndebug1: mux_client_request_session: master session id: 2\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
<28> ESTABLISH SSH CONNECTION FOR USER: None
<28> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o 'ControlPath="/Users/ben/.ansible/cp/d4e83c6eea"' 28 '/bin/sh -c '"'"'( umask 77 && mkdir -p "` echo /u/bencox/.ansible/tmp `"&& mkdir "` echo /u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912 `" && echo ansible-tmp-1687356148.3595898-2931-172301312405912="` echo /u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912 `" ) && sleep 0'"'"''
<28> (0, b'ansible-tmp-1687356148.3595898-2931-172301312405912=/u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912\n', b'OpenSSH_9.0p1, LibreSSL 3.3.6\r\ndebug1: Reading configuration data /Users/ben/.ssh/config\r\ndebug1: /Users/ben/.ssh/config line 1: Applying options for *\r\ndebug1: /Users/ben/.ssh/config line 31: Applying options for 28\r\ndebug1: /Users/ben/.ssh/config line 41: Applying options for *\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 21: include /etc/ssh/ssh_config.d/* matched no files\r\ndebug1: /etc/ssh/ssh_config line 54: Applying options for *\r\ndebug1: Authenticator provider $SSH_SK_PROVIDER did not resolve; disabling\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 2852\r\ndebug3: mux_client_request_session: session request sent\r\ndebug1: mux_client_request_session: master session id: 2\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
Loading collection ibm.ibm_zos_core from /Users/ben/Downloads/try-ibm_zos_cics-1.1.0-beta.1/playbooks/collections/ansible_collections/ibm/ibm_zos_core
Using module file /Users/ben/Downloads/try-ibm_zos_cics-1.1.0-beta.1/playbooks/collections/ansible_collections/ibm/ibm_zos_cics/plugins/modules/global_catalog.py
<28> PUT /Users/ben/.ansible/tmp/ansible-local-2828rh5eaoc9/tmp_t9evusu TO /u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/AnsiballZ_global_catalog.py
<28> SSH: EXEC sftp -b - -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o 'ControlPath="/Users/ben/.ansible/cp/d4e83c6eea"' '[28]'
<28> (0, b'sftp> put /Users/ben/.ansible/tmp/ansible-local-2828rh5eaoc9/tmp_t9evusu /u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/AnsiballZ_global_catalog.py\n', b'OpenSSH_9.0p1, LibreSSL 3.3.6\r\ndebug1: Reading configuration data /Users/ben/.ssh/config\r\ndebug1: /Users/ben/.ssh/config line 1: Applying options for *\r\ndebug1: /Users/ben/.ssh/config line 31: Applying options for 28\r\ndebug1: /Users/ben/.ssh/config line 41: Applying options for *\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 21: include /etc/ssh/ssh_config.d/* matched no files\r\ndebug1: /etc/ssh/ssh_config line 54: Applying options for *\r\ndebug1: Authenticator provider $SSH_SK_PROVIDER did not resolve; disabling\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 2852\r\ndebug3: mux_client_request_session: session request sent\r\ndebug1: mux_client_request_session: master session id: 2\r\ndebug2: Remote version: 3\r\ndebug2: Server supports extension "posix-rename@openssh.com" revision 1\r\ndebug2: Server supports extension "statvfs@openssh.com" revision 2\r\ndebug2: Server supports extension "fstatvfs@openssh.com" revision 2\r\ndebug2: Server supports extension "hardlink@openssh.com" revision 1\r\ndebug2: Server supports extension "fsync@openssh.com" revision 1\r\ndebug2: Sending SSH2_FXP_REALPATH "."\r\ndebug3: Sent message fd 3 T:16 I:1\r\ndebug3: SSH2_FXP_REALPATH . -> /u/bencox\r\ndebug3: Looking up /Users/ben/.ansible/tmp/ansible-local-2828rh5eaoc9/tmp_t9evusu\r\ndebug2: Sending SSH2_FXP_STAT "/u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/AnsiballZ_global_catalog.py"\r\ndebug3: Sent message fd 3 T:17 I:2\r\ndebug1: stat remote: No such file or directory\r\ndebug2: do_upload: upload local "/Users/ben/.ansible/tmp/ansible-local-2828rh5eaoc9/tmp_t9evusu" to remote "/u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/AnsiballZ_global_catalog.py"\r\ndebug2: Sending SSH2_FXP_OPEN "/u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/AnsiballZ_global_catalog.py"\r\ndebug3: Sent dest message SSH2_FXP_OPEN I:3 P:/u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/AnsiballZ_global_catalog.py M:0x001a\r\ndebug3: Sent message SSH2_FXP_WRITE I:5 O:0 S:32768\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 5 32768 bytes at 0\r\ndebug3: Sent message SSH2_FXP_WRITE I:6 O:32768 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:7 O:65536 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:8 O:98304 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:9 O:131072 S:32768\r\ndebug3: Sent message SSH2_FXP_WRITE I:10 O:163840 S:21536\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 6 32768 bytes at 32768\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 7 32768 bytes at 65536\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 8 32768 bytes at 98304\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 9 32768 bytes at 131072\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: In write loop, ack for 10 21536 bytes at 163840\r\ndebug3: Sent message SSH2_FXP_CLOSE I:4\r\ndebug3: SSH2_FXP_STATUS 0\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
<28> ESTABLISH SSH CONNECTION FOR USER: None
<28> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o 'ControlPath="/Users/ben/.ansible/cp/d4e83c6eea"' 28 '/bin/sh -c '"'"'chmod u+x /u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/ /u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/AnsiballZ_global_catalog.py && sleep 0'"'"''
<28> (0, b'', b'OpenSSH_9.0p1, LibreSSL 3.3.6\r\ndebug1: Reading configuration data /Users/ben/.ssh/config\r\ndebug1: /Users/ben/.ssh/config line 1: Applying options for *\r\ndebug1: /Users/ben/.ssh/config line 31: Applying options for 28\r\ndebug1: /Users/ben/.ssh/config line 41: Applying options for *\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 21: include /etc/ssh/ssh_config.d/* matched no files\r\ndebug1: /etc/ssh/ssh_config line 54: Applying options for *\r\ndebug1: Authenticator provider $SSH_SK_PROVIDER did not resolve; disabling\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 2852\r\ndebug3: mux_client_request_session: session request sent\r\ndebug1: mux_client_request_session: master session id: 2\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
<28> ESTABLISH SSH CONNECTION FOR USER: None
<28> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o 'ControlPath="/Users/ben/.ansible/cp/d4e83c6eea"' -tt 28 '/bin/sh -c '"'"'_BPXK_AUTOCVT=ON ZOAU_HOME=/tools/zoautil PYTHONPATH=/tools/zoautil/lib LIBPATH=/tools/zoautil/lib:/python/v3r9/usr/lpp/IBM/cyp/v3r9/pyz/lib:/lib:/usr/lib:. PATH=/tools/zoautil/bin:/python/v3r9/usr/lpp/IBM/cyp/v3r9/pyz/bin:/bin:/var/bin _CEE_RUNOPTS='"'"'"'"'"'"'"'"'FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)'"'"'"'"'"'"'"'"' _TAG_REDIR_ERR=txt _TAG_REDIR_IN=txt _TAG_REDIR_OUT=txt LANG=C /python/v3r9/usr/lpp/IBM/cyp/v3r9/pyz/bin/python3.9 /u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/AnsiballZ_global_catalog.py && sleep 0'"'"''
<28> (1, b'\r\n{"is_zos": true, "starting_catalog": {"size": {"unit": "M", "primary": 5, "secondary": 1}, "name": "BENCOX.REGIONS.mixed.DFHGCD", "sdfhload": "CTS560.CICS730.SDFHLOAD", "state": "initial", "autostart_override": "", "nextstart": "", "exists": false, "vsam": false}, "idcams_output": {"rc": 12, "stdout": "1IDCAMS  SYSTEM SERVICES                                           TIME: 15:02:30        06/21/23     PAGE      1\\n0        \\n     DEFINE CLUSTER -\\n         (NAME(BENCOX.REGIONS.......DFHGCD) -\\n0IDC3203I ITEM \'BENCOX.REGIONS......\' DOES NOT ADHERE TO RESTRICTIONS\\n0        INDEXED                      -\\n         MEGABYTES(5 1)             -\\n         SHR(2)              -\\n         FREESPACE(10 10)              -\\n         RECORDSIZE(4089 32760)       -\\n         REUSE)              -\\n         DATA                           -\\n         (NAME(BENCOX.REGIONS.......DFHGCD.DATA)  -\\n         CONTROLINTERVALSIZE(32768)    -\\n         KEYS(52 0))  -\\n         INDEX                          -\\n         (NAME(BENCOX.REGIONS.......DFHGCD.INDEX))\\n0IDC3202I ABOVE TEXT BYPASSED UNTIL NEXT COMMAND. CONDITION CODE IS 12\\n0        \\n         \\n0IDC0002I IDCAMS PROCESSING COMPLETE. MAXIMUM CONDITION CODE WAS 12\\n", "stderr": ""}, "failed": true, "msg": "IDCAMS failed with rc 12 and message: 1IDCAMS  SYSTEM SERVICES                                           TIME: 15:02:30        06/21/23     PAGE      1\\n0        \\n     DEFINE CLUSTER -\\n         (NAME(BENCOX.REGIONS.......DFHGCD) -\\n0IDC3203I ITEM \'BENCOX.REGIONS......\' DOES NOT ADHERE TO RESTRICTIONS\\n0        INDEXED                      -\\n         MEGABYTES(5 1)             -\\n         SHR(2)              -\\n         FREESPACE(10 10)              -\\n         RECORDSIZE(4089 32760)       -\\n         REUSE)              -\\n         DATA                           -\\n         (NAME(BENCOX.REGIONS.......DFHGCD.DATA)  -\\n         CONTROLINTERVALSIZE(32768)    -\\n         KEYS(52 0))  -\\n         INDEX                          -\\n         (NAME(BENCOX.REGIONS.......DFHGCD.INDEX))\\n0IDC3202I ABOVE TEXT BYPASSED UNTIL NEXT COMMAND. CONDITION CODE IS 12\\n0        \\n         \\n0IDC0002I IDCAMS PROCESSING COMPLETE. MAXIMUM CONDITION CODE WAS 12\\n", "invocation": {"module_args": {"state": "initial", "sdfhload": "CTS560.CICS730.SDFHLOAD", "location": "BENCOX.REGIONS.mixed.DFHGCD", "space_primary": 5, "space_type": "M"}}}\r\n', b'OpenSSH_9.0p1, LibreSSL 3.3.6\r\ndebug1: Reading configuration data /Users/ben/.ssh/config\r\ndebug1: /Users/ben/.ssh/config line 1: Applying options for *\r\ndebug1: /Users/ben/.ssh/config line 31: Applying options for 28\r\ndebug1: /Users/ben/.ssh/config line 41: Applying options for *\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 21: include /etc/ssh/ssh_config.d/* matched no files\r\ndebug1: /etc/ssh/ssh_config line 54: Applying options for *\r\ndebug1: Authenticator provider $SSH_SK_PROVIDER did not resolve; disabling\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 2852\r\ndebug3: mux_client_request_session: session request sent\r\ndebug1: mux_client_request_session: master session id: 2\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 1\r\nShared connection to winmvs28.hursley.ibm.com closed.\r\n')
<28> Failed to connect to the host via ssh: OpenSSH_9.0p1, LibreSSL 3.3.6
debug1: Reading configuration data /Users/ben/.ssh/config
debug1: /Users/ben/.ssh/config line 1: Applying options for *
debug1: /Users/ben/.ssh/config line 31: Applying options for 28
debug1: /Users/ben/.ssh/config line 41: Applying options for *
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 21: include /etc/ssh/ssh_config.d/* matched no files
debug1: /etc/ssh/ssh_config line 54: Applying options for *
debug1: Authenticator provider $SSH_SK_PROVIDER did not resolve; disabling
debug1: auto-mux: Trying existing master
debug2: fd 3 setting O_NONBLOCK
debug2: mux_client_hello_exchange: master version 4
debug3: mux_client_forwards: request forwardings: 0 local, 0 remote
debug3: mux_client_request_session: entering
debug3: mux_client_request_alive: entering
debug3: mux_client_request_alive: done pid = 2852
debug3: mux_client_request_session: session request sent
debug1: mux_client_request_session: master session id: 2
debug3: mux_client_read_packet: read header failed: Broken pipe
debug2: Received exit status from master 1
Shared connection to winmvs28.hursley.ibm.com closed.
<28> ESTABLISH SSH CONNECTION FOR USER: None
<28> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o 'ControlPath="/Users/ben/.ansible/cp/d4e83c6eea"' 28 '/bin/sh -c '"'"'rm -f -r /u/bencox/.ansible/tmp/ansible-tmp-1687356148.3595898-2931-172301312405912/ > /dev/null 2>&1 && sleep 0'"'"''
<28> (0, b'', b'OpenSSH_9.0p1, LibreSSL 3.3.6\r\ndebug1: Reading configuration data /Users/ben/.ssh/config\r\ndebug1: /Users/ben/.ssh/config line 1: Applying options for *\r\ndebug1: /Users/ben/.ssh/config line 31: Applying options for 28\r\ndebug1: /Users/ben/.ssh/config line 41: Applying options for *\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 21: include /etc/ssh/ssh_config.d/* matched no files\r\ndebug1: /etc/ssh/ssh_config line 54: Applying options for *\r\ndebug1: Authenticator provider $SSH_SK_PROVIDER did not resolve; disabling\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 2852\r\ndebug3: mux_client_request_session: session request sent\r\ndebug1: mux_client_request_session: master session id: 2\r\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 0\r\n')
fatal: [winmvs28]: FAILED! => {
    "changed": false,
    "idcams_output": {
        "rc": 12,
        "stderr": "",
        "stdout": "1IDCAMS  SYSTEM SERVICES                                           TIME: 15:02:30        06/21/23     PAGE      1\n0        \n     DEFINE CLUSTER -\n         (NAME(BENCOX.REGIONS.......DFHGCD) -\n0IDC3203I ITEM 'BENCOX.REGIONS......' DOES NOT ADHERE TO RESTRICTIONS\n0        INDEXED                      -\n         MEGABYTES(5 1)             -\n         SHR(2)              -\n         FREESPACE(10 10)              -\n         RECORDSIZE(4089 32760)       -\n         REUSE)              -\n         DATA                           -\n         (NAME(BENCOX.REGIONS.......DFHGCD.DATA)  -\n         CONTROLINTERVALSIZE(32768)    -\n         KEYS(52 0))  -\n         INDEX                          -\n         (NAME(BENCOX.REGIONS.......DFHGCD.INDEX))\n0IDC3202I ABOVE TEXT BYPASSED UNTIL NEXT COMMAND. CONDITION CODE IS 12\n0        \n         \n0IDC0002I IDCAMS PROCESSING COMPLETE. MAXIMUM CONDITION CODE WAS 12\n"
    },
    "invocation": {
        "module_args": {
            "location": "BENCOX.REGIONS.mixed.DFHGCD",
            "sdfhload": "CTS560.CICS730.SDFHLOAD",
            "space_primary": 5,
            "space_type": "M",
            "state": "initial"
        }
    },
    "is_zos": true,
    "msg": "IDCAMS failed with rc 12 and message: 1IDCAMS  SYSTEM SERVICES                                           TIME: 15:02:30        06/21/23     PAGE      1\n0        \n     DEFINE CLUSTER -\n         (NAME(BENCOX.REGIONS.......DFHGCD) -\n0IDC3203I ITEM 'BENCOX.REGIONS......' DOES NOT ADHERE TO RESTRICTIONS\n0        INDEXED                      -\n         MEGABYTES(5 1)             -\n         SHR(2)              -\n         FREESPACE(10 10)              -\n         RECORDSIZE(4089 32760)       -\n         REUSE)              -\n         DATA                           -\n         (NAME(BENCOX.REGIONS.......DFHGCD.DATA)  -\n         CONTROLINTERVALSIZE(32768)    -\n         KEYS(52 0))  -\n         INDEX                          -\n         (NAME(BENCOX.REGIONS.......DFHGCD.INDEX))\n0IDC3202I ABOVE TEXT BYPASSED UNTIL NEXT COMMAND. CONDITION CODE IS 12\n0        \n         \n0IDC0002I IDCAMS PROCESSING COMPLETE. MAXIMUM CONDITION CODE WAS 12\n",
    "starting_catalog": {
        "autostart_override": "",
        "exists": false,
        "name": "BENCOX.REGIONS.mixed.DFHGCD",
        "nextstart": "",
        "sdfhload": "CTS560.CICS730.SDFHLOAD",
        "size": {
            "primary": 5,
            "secondary": 1,
            "unit": "M"
        },
        "state": "initial",
        "vsam": false
    }
}

### Ansible configuration.

_No response_

### Contents of the inventory

_No response_

### Contents of `group_vars` or `host_vars`

_No response_

### PR DESCRIPTION
**Pull request by AndreMarcel99**
##### SUMMARY
Fixes #822 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
Add .upper() to the command use for idcams
Captura de pantalla 2024-06-25 a la(s) 2 08 15 p m
Captura de pantalla 2024-06-25 a la(s) 2 08 28 p m
Captura de pantalla 2024-06-27 a la(s) 10 02 07 a m
Captura de pantalla 2024-06-27 a la(s) 10 02 16 a m
Captura de pantalla 2024-06-27 a la(s) 2 32 14 p m
Captura de pantalla 2024-06-27 a la(s) 2 32 23 p m
Captura de pantalla 2024-06-27 a la(s) 2 46 07 p m
Captura de pantalla 2024-06-27 a la(s) 2 46 16 p m

### CODE REVIEW

**Review by fernandofloresg — CHANGES_REQUESTED** [review]
Would like to see a test case added, else how would we know it is working now?

**Review by richp405 — APPROVED** [review]

File: changelogs/fragments/1550-lower_case_idcams_utility.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+trivial:
+  - zos_operator - Pass to capital letters the command to idcams utility.
```
Comment (rexemin) [inline changelogs/fragments/1550-lower_case_idcams_utility.yml:2]: Would rephrase this to `Pass capital letters to the command...`

**Comment by AndreMarcel99** [comment]
@fernandofloresg I just modify zos find test case to validate by calling a case is calling the function

**Review by fernandofloresg — APPROVED** [review]
